File: backend/Core/DTOs/SharedListPreviewDto.cs
namespace Core.DTOs
{
    public sealed class SharedListPreviewDto
    {
        public Guid ShareToken { get; set; }
        public string ListName { get; set; } = null!;
        public string OwnerName { get; set; } = null!;
        public int PerfumeCount { get; set; }
        public IReadOnlyList<string> PreviewImages { get; set; } = [];
    }
}

File: mobile/src/app/models/sharedlistpreview.dto.ts
import { SharedListPerfumePreviewDto } from './sharedlistperfumepreview.dto';

export interface SharedListPreviewDto {
  shareToken: string;
  listName: string;
  ownerName: string;
  perfumeCount: number;
  perfumes: SharedListPerfumePreviewDto[];
}
